sklearn/tree/tests/test_tree.py

```diff
@@ -1237,7 +1240,8 @@ def test_arrays_persist():
     # non-regression for #2726
     for attr in ['n_classes', 'value', 'children_left', 'children_right',
                  'threshold', 'impurity', 'feature', 'n_node_samples']:
-        value = getattr(DecisionTreeClassifier().fit([[0], [1]], [0, 1]).tree_, attr)
+        value = getattr(DecisionTreeClassifier()
```

Comment:
Please refrain from making changes that are not related to the main topic of your PR. This adds noise and makes the PR review less pleasant for everyone involved.